generate.py

```diff
@@ -213,7 +213,7 @@ def normalize(code):
       base_code = base_language_code(normalized_code)
       if base_code in language['codes']:
         language_name = language.get('names', [None])[0]
-        language_slug = slugify(language_name) if language_name else code
+        language_slug = slugify(language_name) if language_name else base_code
```

Comment:
What does this change mean?  For which languages does it change something?  e.g Chinese?

Reply:
At the moment, the change doesn't make any difference since I have also added the names for 2-letter codes. However, I just changed it to ```base_code``` since the validation was done with ```base_code```.